docs/README.md

```diff
@@ -0,0 +1,22 @@
+# Contribute to Documentation
+
+## Local Setup
+The current documentation is built using Continuous Integration (CI). You can also build the documentation locally following these steps:
+
+```bash
+# Make sure you are at the root of the repository before executing these commands
+python -m pip install --upgrade pip
+pip install -e .[tf]
+pip install -e .[docs]
+```
+
+Now you can build the documentation by running:
+```bash
+make html
```

Comment:
We don't have `make html` it's `make docs-single-page`

Reply:
What do you mean ? `make html` works on my side :sweat_smile:

Reply:
Mh we don't have a command html in Makefile 😅

Reply:
How can this work xD docs-single-version builds only the actual docs (for testing)

Reply:
Because we have the following `Makefile`:
```
# Catch-all target: route all unknown targets to Sphinx using the new
# "make mode" option.  $(O) is meant as a shortcut for $(SPHINXOPTS).
%: Makefile
	@$(SPHINXBUILD) -M $@ "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
```

`html` is forwarded to `sphinx-build` where `html` is a subcommand, that's why it works

Reply:
ok then we should decide on a command and adapt the Makefile accordingly 👍

Reply:
Ok, so I just checked both command. `make html` invokes `sphinx-build` with builder named `html`: https://www.sphinx-doc.org/en/master/man/sphinx-build.html.
But `html` is the default builder, so `make html` here or `make docs-single-version` at root repository produces exactly the same thing. So let's keep `make docs-single-version`